src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -383,6 +384,8 @@ def load_arguments(self, _):
         c.argument('disk_driver_version', arg_type=get_enum_type(disk_driver_versions))
         c.argument('disable_disk_driver', action='store_true')
         c.argument('enable_file_driver', action='store_true')
+        c.argument('enable_namespace_resources', help='Enables sync of namespaces as Azure Resource Manager resources')
+        c.argument('disable_namespace_resources', help='Disables sync of namespaces as Azure Resource Manager resources')
```

Comment:
```suggestion
        c.argument('enable_namespace_resources', help='Enable sync of namespaces as Azure Resource Manager resources')
        c.argument('disable_namespace_resources', help='Disable sync of namespaces as Azure Resource Manager resources')
```

Reply:
Could we define those parameters as action='store_true'?

Reply:
@FumingZhang May I ask is the design of paired bool parameters in line with the expectations of the AKS team?

Reply:
Sorry for the late reply. I would say yes. PM from AKS team would like to keep using paired bool parameters.